content/en/blog/_posts/2022-04-28-cni-version-related-breaking-change.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: blog
+title: "CNI version-related breaking change in Kubernetes 1.24 GA"
+date: 2022-04-28T12:00:00+0000
+slug: cni-version-related-breaking-change-k8s-1.24
+canonicalUrl: https://kubernetes.dev/blog/2022/04/28/cni-version-related-breaking-change-k8s-1.24/
+---
+
+**Authors & Contributors:** [Mike Brown](https://github.com/mikebrow), [Rolfe Dlugy-Hegwer](https://github.com/rolfedh),[Nate W.](https://github.com/nate-double-u), and others.
```

Comment:
```suggestion
**Authors:** Mike Brown, Rolfe Dlugy-Hegwer, Nate Waddington, and others.
```

Usually we list authors' employers / affiliations after their names, inside parens.

Reply:
If we remove the _contributors_ designation, I'm not sure I belong in the list -- at best I think my byline would be "editor," and even then I'm happy to be a part of the _, and others_ here :)

Reply:
Updated. Thanks.